src/plugins/discover/public/application/components/default_discover_table/table_row.tsx

```diff
@@ -111,7 +111,11 @@ const TableRowUI = ({
             <td
               key={colName}
               data-test-subj="docTableField"
-              className="osdDocTableCell eui-textBreakAll eui-textBreakWord"
+              className={`osdDocTableCell ${
+                indexPattern.timeFieldName === colName
```

Comment:
Out of curiosity, will there ever be a case where `indexPattern.timeFieldName !== colName`?

Reply:
don't think so, there is already another condition which used the same to check if timefield using the same condition
https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/discover/public/application/components/default_discover_table/table_row.tsx#L129

Reply:
> Out of curiosity, will there ever be a case where indexPattern.timeFieldName !== colName?

without diving too deep based on the snippet I'm guessing that this is for all the fields and it was applying the text break all to each colName. For example like if they select timestamp, useriD, statusCode, etc. are all col names. It just timefieldName sticks out because it's the first one we see. Other columns might have a child element that don't let it get wrapped as easily.

To be honest I think this rule could actually be applied to all fields of type `date`. But that shouldn't block this PR.

Reply:
created an issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/8441